flake.nix

```diff
@@ -1,77 +1,144 @@
 {
-  description = "Template for Holochain app development";
+  description = "Flake for Holochain app development";
 
   inputs = {
-    nixpkgs.follows = "holochain/nixpkgs";
-    versions.url = "github:holochain/holochain?dir=versions/weekly";
+    holonix.url = "github:holochain/holonix?ref=main";
 
-    holochain = {
-      url = "github:holochain/holochain";
-      inputs.versions.follows = "versions";
+    nixpkgs.follows = "holonix/nixpkgs";
+    flake-parts.follows = "holonix/flake-parts";
+
+    crane = {
+      url = "github:ipetkov/crane";
+      inputs.nixpkgs.follows = "holonix/nixpkgs";
```

Comment:
Should not be required if you do a flake update, they have removed their nixpkgs input